sources/specUtils.ts

```diff
@@ -96,10 +99,12 @@ export async function loadSpec(initialCwd: string): Promise<LoadSpecResult> {
       continue;
 
     const manifestPath = path.join(currCwd, `package.json`);
-    if (!fs.existsSync(manifestPath))
-      continue;
-
-    const content = await fs.promises.readFile(manifestPath, `utf8`);
+    try {
+      content = await fs.promises.readFile(manifestPath, `utf8`);
+    } catch (err) {
```

Comment:
nit: The `content` declaration should be right above the `try/catch`, so it's clearer where it's assigned
```suggestion
    let content: string;
    try {
      content = await fs.promises.readFile(manifestPath, `utf8`);
    } catch (err) {
```